src/app/dataset-view/dataset.component.ts

```diff
@@ -55,11 +58,58 @@ export class DatasetComponent implements OnInit, AfterContentInit, OnDestroy {
     public currentSchema: DataViewSchema;
     public isMinimizeSearchAdditionalButtons = false;
     public datasetViewType: DatasetViewTypeEnum = DatasetViewTypeEnum.overview;
-    public searchAdditionalButtonsData: SearchAdditionalButtonInterface[] = [
-        {
-            textButton: searchAdditionalButtonsEnum.DeriveFrom,
-        },
-    ];
+    public searchAdditionalButtonsData: SearchAdditionalHeaderButtonInterface[] =
```

Comment:
I think this generic nature of the "additional" buttons is not very helpful. The set of buttons doesn't change and this only makes code more complex. I think in future we should move them to a plain HTML inside of the component.